### ISSUE TITLE
Verification of json file failed because queries were made from IPs originating from AWS-AWSManagedRulesAnonymousIpList

### ISSUE THREAD

**Issue by pjaskierny-wbd** [WICG/first-party-sets#208]
We added out RWS configuration https://github.com/GoogleChrome/related-website-sets/pull/278 and some of your requests failed 403 becacuse your requests were initiated from IPs which are on the AWSManagedRulesAnonymousIpList. This is a security rule that blocks traffic from anonymous IP addresses.

Can we ask for verification from IPs that are not from the "anonymous IPs" list?

**Comment by sjledoux** [comment]
The checks run on a GitHub server, which may be directing its traffic through a proxy and causing this issue. We're looking into a solution now, and we'll update when we find one.

**Comment by pjaskierny-wbd** [comment]
In the meantime, are you able to provide a pool of IP addresses dedicated to performing checks? If this pool is dedicated to you for checks we can add the IP to the exception list until a solution is found on your side.

**Comment by sjledoux** [comment]
Hi, GitHub offers [this list of IPs that its runners use](https://api.github.com/meta). In particular, the "actions" list would be relevant to this case. We also have some internal checks we do, so we may have to check if those cause issues with the AnonymousIP list as well, which we will update you on.

**Comment by sjledoux** [comment]
Hi, have you been able to whitelist those IP addresses? Otherwise, I don't believe we have a long term solution at the moment.

**Comment by pjaskierny-wbd** [comment]
Hi, we are working with allowlist for delivered IPs. I will give update when we will finish it ;-)

**Comment by dgrochowiecki** [comment]
Hello @sjledoux , we are facing similar issue, and creating whitelist rules for more than 4k IP's is not the right solution, because it will significantly increase your WAF costs, is there any other way to identify verification requests? for example header verifificstion etc?

**Comment by plunker2025** [comment]
> We added out RWS configuration https://github.com/GoogleChrome/related-website-sets/pull/278 and some of your requests failed 403 becacuse your requests were initiated from IPs which are on the AWSManagedRulesAnonymousIpList. This is a security rule that blocks traffic from anonymous IP addresses.
> 
> Can we ask for verification from IPs that are not from the "anonymous IPs" list?

**Comment by plunker2025** [comment]
> @sjledoux : any updates about solution on your side? Can you estimate any dates?

Yes  if we can look with Google id developpement can be add solutions to see wen defferent device or Web hook can be retrace with token not cookie something in That way
